.bazelrc.psmdb

```diff
@@ -0,0 +1,100 @@
+################################################################################
+# Percona-specific config flag aliases
+################################################################################
+common --flag_alias=inmemory=//bazel/config:inmemory
+common --flag_alias=audit=//bazel/config:audit
+common --flag_alias=full_featured=//bazel/config:full_featured
+common --flag_alias=enable_fipsmode=//bazel/config:enable_fipsmode
+common --flag_alias=enable_fcbis=//bazel/config:enable_fcbis
+common --flag_alias=enable_oidc=//bazel/config:enable_oidc
+
+################################################################################
+# Default values for Percona-specific build profiles
+################################################################################
+common --enable_platform_specific_config=true
+
+# Disable remote execution and remote cache
+common --remote_executor=
+common --experimental_remote_downloader=
+common --remote_cache=
+common --bes_backend=
+common --bes_results_url=
+common --tls_client_certificate=
+common --tls_client_key=
+common --remote_cache_compression=false
+common --remote_upload_local_results=false
+common --grpc_keepalive_time=0s
+common --legacy_important_outputs
+
+# enable additional modules
+common --//bazel/config:use_sasl_client=True
+common --//bazel/config:inmemory=True
+common --//bazel/config:audit=True
+
+# how to build
+common --build_enterprise=False
```

Comment:
That is the only change I had to make comparing to `v8.0`. @ktrushin I'm not sure why it's not there on `v8.0` ?

Reply:
Hm. [From what I can see](https://github.com/percona/percona-server-mongodb/blob/v8.0/bazel/config/BUILD.bazel#L1863), in `v8.0` this flag is `False` by default and thus doesn't require being explicitly set for PSMDB.
It is strange that the flag is required in `master` because [there](https://github.com/percona/percona-server-mongodb/blob/master/bazel/config/BUILD.bazel#L1840) it is also `False` by default.

Reply:
@ktrushin that is because the main `.bazelrc` file sets the value to `true`, see here:
https://github.com/percona/percona-server-mongodb/blob/746a507950103c3544ca58893907005812edac76/.bazelrc#L226

Reply:
Yes I've seen that removal. However, I thought we can avoid doing that by making sure the default build configuration is properly overwritten in the `.bazelrc.psmdb` file. By keeping the original `.bazelrc` file almost intact and having the only long-lived difference the importing of the `.bazelrc.psmdb` we can make the merging process much easier. In this approach we can also have the upstream's default values to evolve and we would "consume" them as long as we decide not to do so - in such case we would overwrite the values in `.bazelrc.psmdb` file.

Maybe that is because I'm a bit biased after working on the merging recently, however I believe this might be a better solution for us in the long term. What do you think ?